glue/core/roi.py

```diff
@@ -271,12 +338,14 @@ def __gluestate__(self, context):
         return dict(xmin=context.do(self.xmin),
                     xmax=context.do(self.xmax),
                     ymin=context.do(self.ymin),
-                    ymax=context.do(self.ymax))
+                    ymax=context.do(self.ymax),
+                    theta=context.do(self.theta))
 
     @classmethod
     def __setgluestate__(cls, rec, context):
         return cls(xmin=context.object(rec['xmin']), xmax=context.object(rec['xmax']),
-                   ymin=context.object(rec['ymin']), ymax=context.object(rec['ymax']))
+                   ymin=context.object(rec['ymin']), ymax=context.object(rec['ymax']),
+                   theta=context.object(rec['theta']))
```

Comment:
Here we have to take into account theta might not exist in old states/session files, and default to zero.